feg/gateway/services/session_proxy/credit_control/gx/config_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gx
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"magma/feg/cloud/go/protos/mconfig"
+
+	"magma/feg/gateway/services/session_proxy/credit_control"
+
+	managed_configs "magma/gateway/mconfig"
+)
```

Comment:
please use same format and separation as in other files (group them by go modules, public, magma)